app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -703,6 +704,13 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
      */
     private $catalogConfig;
 
+    /**
+     * Stock Item Importer
+     *
+     * @var StockItemImporterInterface $stockItemImporter
```

Comment:
Currently Magento does not use any standard which requires adding variable name in this kind of docblock

Reply:
Done.